test/browser/browser_discovery_render.js

```diff
@@ -8,6 +8,9 @@ async function before({ pushPrefs }) {
       enabled: true,
       hardcoded_layout: true,
     }),
+  ], [
+    "browser.newtabpage.activity-stream.discoverystream.enabled",
+    true,
```

Comment:
Probably could set this in browser.ini as individual tests determine if config.enabled is true or unset:
https://github.com/mozilla/activity-stream/blob/04220818451a4695f32298db08d325a1862892cf/test/browser/browser.ini#L6-L8

Reply:
Trying!

Reply:
Pushing up a second attempt that should get this going.